controllers/apply/expiry.js

```diff
@@ -78,29 +78,41 @@ async function sendExpiryEmails(req, emailQueue, locale) {
 
     return await Promise.all(
         emailQueue.map(async emailToSend => {
+            const getAppData = field =>
+                get(emailToSend.PendingApplication, `applicationData.${field}`);
+
+            const projectCountry = getAppData('projectCountry');
```

Comment:
Maybe inline this as it's only used once

Reply:
It's used again further down for `projectName` (although I'll replace the duplicated calls for `projectCountry`)